ibexa/commerce/4.4/templates/themes/standard/full/welcome_page.html.twig

```diff
@@ -3,156 +3,307 @@
+                </a>
+            </div>
+            <div class="ibexa-welcome-column ibexa-welcome-tutorials-and-resources__block ibexa-welcome-tutorials-and-resources__block--rest-api">
+                <div class="ibexa-welcome-circle">
+                    <img
+                        src="{{ asset('build/images/rest-api.svg') }}"
+                        alt="{{ 'ibexa_platform_welcome_page.rest_api.image'|trans|desc('REST API reference') }}"
+                    />
                 </div>
-                <div class="col-lg-4 col-md-6 mt-lg-0 mt-5">
-                    <div class="d-flex flex-row justify-content-center mr-auto">
-                        <div class="mr-3">
-                            <img src="{{ asset('build/images/conversation-chat-icon.svg') }}" alt="Conversation chat icon">
-                        </div>
-                        <div>
-                            <h2 class="mb-3">{{ 'ibexa_platform_welcome_page.forums_and_community'|trans|desc('Forums and community') }}</h2>
-                            <ul class="list-unstyled references-list">
-                                <li class="references-list-item">
-                                    <a href="https://discuss.ibexa.co" class="references-list-item__link" target="_blank">
-                                        {{ 'ibexa_platform_welcome_page.discussion_forum'|trans|desc('Discussion forum') }}
-                                    </a>
-                                </li>
-                                <li class="references-list-item">
-                                    <a href="https://ez-community-on-slack.herokuapp.com/" class="references-list-item__link" target="_blank">
-                                        {{ 'ibexa_platform_welcome_page.community_slack_channel'|trans|desc('Community Slack channel') }}
-                                    </a>
-                                </li>
-                                <li class="references-list-item">
-                                    <a href="https://ibexa.co/blog" class="references-list-item__link" target="_blank">
-                                        {{ 'ibexa_platform_welcome_page.ibexa_blog'|trans|desc('Ibexa blog') }}
-                                    </a>
-                                </li>
-                            </ul>
-                        </div>
-                    </div>
+                <h3 class="ibexa-welcome-block-header">
+                    {{ 'ibexa_platform_welcome_page.rest_api.title'|trans|desc('REST API reference') }}
+                </h3>
+                <p class="ibexa-welcome-paragraph">
+                    {{ 'ibexa_platform_welcome_page.rest_api.content'|trans|desc('As an API-first product, Ibexa DXP focuses on well-structured APIs. REST API endpoints cover most functionalities of the product.') }}
+                </p>
+                <a
+                    href="http://doc.ibexa.co/rest-api-reference"
```

Comment:
This link results in 404 - unless there's a doc change that's required.
Also, shouldn't we use https by default?